ci-operator/jobs/openstack-k8s-operators/placement-operator/openstack-k8s-operators-placement-operator-main-presubmits.yaml

```diff
@@ -156,6 +156,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )placement-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
In favor of this kuttl job, we usually disable the "build-deploy" only job, since kuttl job already do the same build and deploy steps. To do that, you need to manually change `always_run` to `false`, not here, but in line `98`

Reply:
I was thinking of keeping build-deploy job until kuttl job is proven to be stable.
However we can fix the kuttl job first and then completely replace build-deploy job.

Reply:
I would err on side of caution in this case. One more job shouldn't hurt us. We should create a task for eventually clearing it, under a suitable epic, however.

Reply:
In this case yes, it will consume an extra OCP cluster to run the same steps. The side effect would be jobs waiting and timeouting due to lack of cluster to run those tests.

Reply:
That is a valid point. That being said we shouldn't drop any existing test until the replacement is ready. So I would rather not merge this, before we can be sure about the new job stability.

Reply:
We confirmed kuttl tests pass, so I think we are ready to replace the deploy job by kuttl job. So I'll update the setting to disable the deploy job.